tests/fixtures/identity.sql

```diff
@@ -818,3 +818,4 @@ JSON_OBJECT('x': NULL, 'y': 1 ABSENT ON NULL WITH UNIQUE KEYS)
 JSON_OBJECT('x': 1 RETURNING VARCHAR(100))
 JSON_OBJECT('x': 1 RETURNING VARBINARY FORMAT JSON ENCODING UTF8)
 SELECT if.x
+SELECT MIN(column_name) KEEP (DENSE_RANK FIRST ORDER BY column_name DESC) FROM table_name
```

Comment:
select y keep

does this break?

Reply:
Doesn't break, but I added a test for `sum(y) keep` which did break before. I isolated `KEEP` for oracle only.